README.md

```diff
@@ -65,7 +65,15 @@ Find out the videos here -
 
 ### Series 2 - AAM (An App a Month 🥭)
 
-Coming soon...
+Find out the videos here -
+
+| S.No   |      Topic      |  YouTube link                                                                          |
+|--------|-----------------|----------------------------------------------------------------------------------------|
+| 1      | Project 1       |  https://www.youtube.com/playlist?list=PLGyA74h_S9NqJvvQ7-l9bUHHS6bMCkgo0              |
```

Comment:
Can you add the actual name of the project or playlist instead of Project 1, Project 2, etc?

Reply:
Ok, I am working on it

Reply:
I've updated the changes.